apps/vr-tests-web-components/tsconfig.json

```diff
@@ -10,5 +10,6 @@
     "jsx": "react",
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31496&commitId=426122d5a47767a368d2bbd5b3d76ad67d2d80ee&env=prod&clientType=FLUENTUI)


  <details><summary><b>Popup</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popup.Popup Example Position.chromium.png|846| Changed|
|Popup.Popup Example Rtl.RTL- Shows popup.chromium.png|513| Changed|
|Popup.Popup Example Pointer Margin.chromium.png|1155| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Tooltip Example Pointer Margin.chromium.png|869| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>